contracts/src/rollup/RollupUserLogic.sol

```diff
@@ -30,12 +30,26 @@ abstract contract AbsRollupUserLogic is
         return deployTimeChainId != block.chainid;
     }
 
+    function _validatorIsAfk() internal view returns (bool) {
+        Node memory latestNode = getNodeStorage(latestNodeCreated());
+        if (latestNode.createdAtBlock == 0) return false;
+        // TODO: what is the max time validators can be afk?
```

Comment:
Could eliminate this todo. This seems reasonable

Reply:
```suggestion
```
But note that if we have validator downtime in testnet with shorter confirmPeriodBlocks we might have issue, should we add some constant here?

Reply:
pushed https://github.com/OffchainLabs/nitro/pull/1025/commits/564c031dd07732df9bd19f47461034c435b52d79

Reply:
Nice